test/middleware/spec.back-link.js

```diff
@@ -176,13 +184,23 @@ describe('Back Links', function () {
 
     it('permits whitelisting of steps from a separate form instance', function () {
         req.get.withArgs('referrer').returns('http://example.com/referrer');
+        req.baseUrl = '/referrer';
```

Comment:
A separate form instance here has a good chance of being on a different baseUrl.